app/src/main/java/ch/epfl/sdp/map/MapsActivity.java

```diff
@@ -12,11 +12,18 @@
 import com.google.android.gms.maps.SupportMapFragment;
 
 import ch.epfl.sdp.R;
+import ch.epfl.sdp.game.Game;
+import ch.epfl.sdp.item.Healthpack;
+import ch.epfl.sdp.item.Item;
 import ch.epfl.sdp.item.Scan;
 
 public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
     public static final MapApi mapApi = new GoogleMapApi();
 
+    public static MapApi getMapApi() {
+        return mapApi;
+    }

```

Comment:
Since mapApi is already public, do we need this getter function?

Reply:
I tried to use this to mock the static mapapi, but I will remove this (it came back after swapping between commits to check leaderboard). Thank you for telling me :)